core/app/helpers/spree/base_helper.rb

```diff
@@ -28,7 +28,7 @@ def meta_data
       object = instance_variable_get('@' + controller_name.singularize)
       meta = {}
 
-      if object.is_a? ActiveRecord::Base
+      if object.is_a? Spree::ApplicationRecord
```

Comment:
I think it is fine to keep it as is. Maybe the store has a custom model that also is an `ActiveRecord` and implements `meta_description`?